drivers/hwinfo/Kconfig

```diff
@@ -0,0 +1,37 @@
+# Kconfig -HW Info driver configuration options
+
+#
+# Copyright (c) 2019 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig HWINFO
+	bool "hwinfo driver"
+	help
+	  Enable hwinfo driver.
+
+if HWINFO
+
+config HWINFO_DEV_NAME
+	string "hwinfo device name"
+	default "HWINFO"
+	help
+	  Specify the device name to be used for thr hwinfo driver.
+
+config HWINFO_DEVICE_ID_STM32
```

Comment:
As per earlier discussion this option enables the whole hwinfo driver for STM32 series. At the moment it happens to provide device id function but it may change in the future. The option should be renamed to `HWINFO_STM32`. The requirement should become clear when looking at drivers/hwinfo/CMakeLists.txt.

Reply:
good point. Changed to HWINFO_STM32